common/src/main/java/io/netty/util/internal/logging/Log4J2LoggerFactory.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2016 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.util.internal.logging;
+
+import org.apache.logging.log4j.LogManager;
+
+public final class Log4J2LoggerFactory extends InternalLoggerFactory {
```

Comment:
I think we should at least keep the public constructor to not break stuff as while its in the internal package user tend to set a logger

> Am 31.03.2016 um 07:46 schrieb Trustin Lee <notifications@github.com>:
> 
> In common/src/main/java/io/netty/util/internal/logging/Log4J2LoggerFactory.java:
> 
> > + * version 2.0 (the "License"); you may not use this file except in compliance
> > + * with the License. You may obtain a copy of the License at:
> > + *
> > + *   http://www.apache.org/licenses/LICENSE-2.0
> > + *
> > + * Unless required by applicable law or agreed to in writing, software
> > + * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
> > + * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
> > + * License for the specific language governing permissions and limitations
> > + * under the License.
> > + */
> > +package io.netty.util.internal.logging;
> > +
> > +import org.apache.logging.log4j.LogManager;
> > +
> > +public final class Log4J2LoggerFactory extends InternalLoggerFactory {
> Making them all final singleton.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
>